src/tcpconnect/main.rs

```diff
@@ -0,0 +1,270 @@
+use bcc::core::BPF;
+use std::net::{Ipv4Addr, Ipv6Addr};
+use clap::{App, Arg};
+use failure::Error;
+use std::process;
+use core::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
+use std::{ptr, str};
+extern crate ctrlc;
```

Comment:
I don't believe this is needed.

Reply:
I followed some of the other examples. What would be problematic about using these?

Reply:
It's the `extern crate ctrlc;` that I think can be removed. We're not using any macros from it, and extern crate is typically not needed in Rust 2018 edition.